_data/glossary/kubernetes-api.yaml

```diff
@@ -0,0 +1,10 @@
+id: kubernetes-api
+name: Kubernetes API
+tags:
+- fundamental
+- architecture
+short-description: |
+  The foundation for how Kubernetes represents and stores the state of the cluster.
```

Comment:
This doesn't actually describe what an apiserver is.

"The application which serves the Kubernetes HTTP API, and which stores the state of the cluster".

Reply:
I'm going with "The application which serves the Kubernetes HTTP API and stores the state of the cluster."